src/server/req_modify.c

```diff
@@ -812,7 +819,7 @@ req_modifyReservation(struct batch_request *preq)
 		switch (index) {
 			case RESV_ATR_start:
 				if ((presv->ri_wattr[RESV_ATR_state].at_val.at_long != RESV_RUNNING) ||
-					!(presv->ri_qp->qu_numjobs)) {
+					!(num_jobs)) {
```

Comment:
Could you remove the () around num_jobs.  I don't think it makes anything clearer.